emulator/src/extensions/replay/ext_replay.c

```diff
@@ -57,9 +57,10 @@ typedef enum
     SCREENSHOT,
     SET_MODE,
     RANDOM_SEED,
+    COMMAND,
 } replayLogType_t;
 
-#define LAST_TYPE RANDOM_SEED
+#define LAST_TYPE COMMAND
```

Comment:
You could add `NUM_REPLAY_LOG_TYPES` as the final enum value and then use that as your loop bound. It saves you having to update `LAST_TYPE` if/when you add more commands.